packages/docs/src/examples/data-tables/intermediate/expand.vue

```diff
@@ -16,8 +16,8 @@
         <v-switch v-model="singleExpand" label="Single expand" class="mt-2"></v-switch>
       </v-toolbar>
     </template>
-    <template v-slot:expanded-item="{ headers }">
-      <td :colspan="headers.length">Peek-a-boo!</td>
+    <template v-slot:expanded-item="{ headers, item }">
+      <td :colspan="headers.length">Peek-a-boo! {{ item.info }}</td>
```

Comment:
should be `item.other`

Reply:
need to add `other` to the remainder of the items in `data`. also remove the `Peek-a-boo!`